### DISCUSSION TITLE
How does `Conductor<IScreen>.Collection.OneActive.ActivateItem` work?

### DISCUSSION THREAD

**Discussion by ShuhuaGao** [canton7/Stylet#261] [Q&A] [answered]
From the [wiki](https://github.com/canton7/Stylet/wiki/Screens-and-Conductors#conductortcollectiononeactive)
> Calling ActivateItem will add the item passed to the Items collection, and will also activate it and set it as the ActiveItem

I then check the source code to learn about the internals. However, I cannot figure out where the item is added to `Items`. Supposing I have a **new** viewmodel instance `item` and try `ActivateItem(item)`, the source code should go to `ChangeActiveItem`. However, inside the `ChangeActiveItem` method, I did not see how `item` was added to `Items`. Could you give some explanation?

Besides, consider the following snippet
```cs
// item is a view model instance
Items.Add(item);
ActivateItem(item);
```
Will `item` be added twice? If not, where is the repetition checked? Thank you.

**Comment by canton7** [comment, marked as answer]
[Here](https://github.com/canton7/Stylet/blob/5e97f9e1ea8c45e9d7033e6e7c7ea8ce41c3420a/Stylet/ConductorOneActive.cs#L216)

**Reply by ShuhuaGao** [reply]
Thanks for the quick reply. My mistake. I did not notice the `override` and actually referred to the base version.
